rfc/5/index.md

```diff
@@ -9,495 +9,645 @@ responses/index
-is indicated by the length of its "axes" array. The "volume_micrometers" example coordinate system above is three dimensional (3D).
-
-The axes of a coordinate system (see below) give information about the types, units, and other properties of the coordinate
-system's dimensions. Axis `name`s may contain semantically meaningful information, but can be arbitrary. As a result, two
-coordinate systems that have identical axes in the same order may not be "the same" in the sense that measurements at the same
-point refer to different physical entities and therefore should not be analyzed jointly. Tasks that require images, annotations,
-regions of interest, etc., SHOULD ensure that they are in the same coordinate system (same name, with identical axes) or can be
-transformed to the same coordinate system before doing analysis. See the example below.
+````{admonition} Example
 
+Coordinate Systems metadata example
 
-### "axes" metadata
-
-"axes" describes the dimensions of a coordinate systems. It is a list of dictionaries, where each dictionary describes a dimension (axis) and:
-- MUST contain the field "name" that gives the name for this dimension. The values MUST be unique across all "name" fields.
-- SHOULD contain the field "type". It SHOULD be one of the strings "array", "space", "time", "channel", "coordinate", or "displacement" but MAY take other string values for custom axis types that are not part of this specification yet.
-- MAY contain the field "discrete". The value MUST be a boolean, and is `true` if the axis represents a discrete dimension.
-- SHOULD contain the field "unit" to specify the physical unit of this dimension. The value SHOULD be one of the following strings, which are valid units according to UDUNITS-2.
+```json
+{
+    "name" : "volume_micrometers",
+    "axes" : [
+        {"name": "z", "type": "space", "unit": "micrometer"},
+        {"name": "y", "type": "space", "unit": "micrometer"},
+        {"name": "x", "type": "space", "unit": "micrometer"}
+    ]
+}
+```
+````
+
+The axes of a coordinate system (see below) give information
+about the types, units, and other properties of the coordinate system's dimensions.
+Axis names may contain semantically meaningful information, but can be arbitrary.
+As a result, two coordinate systems that have identical axes in the same order 
+may not be "the same" in the sense that measurements at the same point
+refer to different physical entities and therefore should not be analyzed jointly.
+Tasks that require images, annotations, regions of interest, etc.,
+SHOULD ensure that they are in the same coordinate system (same name and location within the Zarr hierarchy, with identical axes)
+or can be transformed to the same coordinate system before doing analysis.
+See the [example below](example:coordinate_transformation).
+
+#### "axes" metadata
+
+"axes" describes the dimensions of a coordinate systems
+and adds an interpretation to the samples along that dimension.
```

Comment:
We are defining a list of objects first, and then describing the properties of the elements of the list. IMO it's more logical to define the axis object first, and then define `axes` as a list of axis objects.